include/h2o.h

```diff
@@ -344,6 +344,10 @@ struct st_h2o_globalconf_t {
      * setuid user (or NULL)
      */
     char *user;
+    /**
+     * number of cpus
+     */
+    size_t num_procs;
```

Comment:
I think this should better be a lazily-initialized global variable (or a file-level static variable) that is initialized alongside `map_fd`.